ext/lib/mgmt/mcumgr/samples/smp_svr/zephyr/src/main.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <assert.h>
+#include <zephyr.h>
+#include <string.h>
+#include <stdlib.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt.h>
+#include "logging/mdlog.h"
+#include "logging/reboot_log.h"
+#include "fcb.h"
+#include "stats.h"
+#include "mgmt/smp_bt.h"
+#include "mgmt/buf.h"
+
+#ifdef CONFIG_MCUMGR_CMD_FS_MGMT
+#include "fs_mgmt/fs_mgmt.h"
```

Comment:
Use < include instead.

In Zephyr we use  " to communicate to the reader that this is a relative path from the source file, and < to communicate that the path is relative to some include path.

Reply:
OK, makes sense.  I'm sort of a stickler about <headers belonging to the implementation> and "everything else", even to the point where I flout the local coding style!  I have fixed it.